.nvmrc

```diff
@@ -1 +0,0 @@
-0.10.40
```

Comment:
This should stay. It's useful for people to easily switch between node versions using `nvm`